### PR TITLE
Fixed exclude and change deprecation warning

### PR DESCRIPTION
**Pull request by sergivillar**
This PR fixed an issue you have in the exclude function and also a deprecation warning.

### CODE REVIEW

**Comment by sergivillar** [comment]
Please @MQuy could you take a look at this PR? with the current bug I can't exclude more than one file

File: src/detect.js [hunk 1]
```diff
@@ -18,15 +18,15 @@ function detectDeadCode(compilation, options) {
...
+    .map(pattern => path.resolve(context, pattern))
+    .concat(exclude.map(pattern => `!${pattern}`));
 }
 
 function getUsedExportMap(compilation) {
   const unusedExportMap = {};
 
   compilation.chunks.forEach(function(chunk) {
-    chunk.forEachModule(function(module) {
+    for(const module of chunk.modulesIterable) {
```
Comment (MQuy) [inline src/detect.js]: @sergivillar Is there any reason to make this change? Will it work for both webpack 3 and wepback 4 🤔

**Comment by MQuy** [comment]
👍🏻 I will try in Webpack 3 to see if there are any issues.

**Comment by sergivillar** [comment]
thanks @MQuy , can you make a bump version when it's possible??

**Comment by MQuy** [comment]
just tested, it worked fine. Bumped 👍🏻

**Pull request merged**